src/fmu/dataio/_export_item.py

```diff
@@ -234,13 +264,31 @@ def _data_process_content(self):
             usecontent = (list(content.keys()))[0]
             useextra = content[usecontent]
 
-        if usecontent not in ALLOWED_CONTENTS:
-            raise ValueError(f"Sorry, content <{usecontent}> is not in list!")
+        if usecontent not in ALLOWED_CONTENTS.keys():
+            raise ValidationError(f"Invalid content: <{usecontent}> is not in list!")
```

Comment:
Suggest changing "in list" to "not in the list of allowed content" or something. I think I would be wondering "which list...? 🤔" if I received this message.

Reply:
Will fix